app/src/main/java/org/mozilla/focus/screenshot/ScreenshotViewerActivity.java

```diff
@@ -122,13 +122,12 @@ protected void onCreate(Bundle savedInstanceState) {
                         requestPermissions(new String[]{Manifest.permission.READ_EXTERNAL_STORAGE}, REQUEST_CODE_VIEW_SCREENSHOT);
                     }
                 }
+                showProgressBar(DELAY_MILLIS_TO_SHOW_PROGRESS_BAR);
```

Comment:
Progress bar will still show after user denied permission.
I think move showProgressBar() to initScreenshotInfo() can solve this problem.